onnx/backend/test/case/node/cast.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import onnx
+from onnx import TensorProto
+from onnx.mapping import TENSOR_TYPE_TO_NP_TYPE as type_map
+
+from ..base import Base
+from . import expect
+
+
+class Cast(Base):
+
+    @staticmethod
+    def export():
+        shape = (3, 4)
+        test_cases = [
+            ('FLOAT', 'FLOAT16'),
+            ('FLOAT', 'DOUBLE'),
+            ('FLOAT16', 'FLOAT'),
+            ('FLOAT16', 'DOUBLE'),
+            ('DOUBLE', 'FLOAT'),
+            ('DOUBLE', 'FLOAT16'),
+        ]   
+
+        for case in test_cases:
+            from_type = case[0]
+            to_type = case[1]
+            input = np.random.random_sample(shape).astype(type_map[getattr(TensorProto, from_type)])
```

Comment:
In the generated doc, user may be confused about the type_map. If we put TENSOR_TYPE_TO_NP_TYPE here directly, it will be clear to users.

Reply:
Sure, pushed an update.